docs/src/main/sphinx/connector/faker.md

```diff
@@ -126,11 +125,13 @@ See the Datafaker's documentation for more information about
 [the expression](https://www.datafaker.net/documentation/expressions/) syntax
 and [available providers](https://www.datafaker.net/documentation/providers/).
 
-To test a generator expression, without having to recreate the table, use the
-`random_string` function from the `default` schema:
+Use the `random_string` function from the `default` schema of the `generator`
+catalog to test a generator expression:
 
 ```sql
-SELECT default.random_string('#{Name.first_name}')
```

Comment:
I am not sure if this actually works but in either case .. it relies on a default catalog being set and is therefore to implicit.

Reply:
Let's only keep one example here - remove the second one starting with `USE`.

I tested this locally with:
```bash
docker run -d --name trino --rm -e CATALOG_MANAGEMENT=dynamic -p8080:8080 trino:465-SNAPSHOT-arm64
trino --execute "create catalog generator using faker; select generator.default.random_string('#{currency.code}');"
```

Which prints:
```
CREATE CATALOG
 _col0 
-------
 XBB   
(1 row)
```

Reply:
Done